src/ontology/metadata/omim.yml

```diff
@@ -28,4 +28,4 @@ description: >
 homepage: https://www.omim.org/
 base_prefix_map:
   OMIM: https://omim.org/entry/
-  OMIMPS: https://www.omim.org/phenotypicSeries/PS
```

Comment:
## The primary changes
6 instances of removing `www.` from the OMIMPS URI prefix was done.

Basically I did a global find, found that some files with certain extensions (filter: `!*.owl,!*.ipynb,!*.ttl,!*.tsv`) ought not have the replacement. All other instances I saw seemed like they ought to have the replacement.

The particular change highlighted by this review comment is the one that fixed the missing OMIM slurp parents issue.